pt/lessons/advanced/otp-supervisors.md

```diff
@@ -1,5 +1,5 @@
 ---
-version: 1.1.1
+version: 1.1.2
```

Comment:
```suggestion
version: 1.1.1
```

@AdrielBento The version here refers to the version of the original text in english. It should only be bumped if we are updating the translation with new content from the original lesson.

As you are only fixing a typo, the version should remain the same.